frontend/src/metabase/query_builder/components/DataSelector/DataSelector.jsx

```diff
@@ -197,6 +198,7 @@ export class UnconnectedDataSelector extends Component {
     isPopover: true,
     isMantine: false,
     canSelectMetric: false,
+    canSelectSavedQuestion: true,
```

Comment:
Since this is a new prop, we always show `saved questions` first, so I default it to `true` here. We actually might not need this because I suspect `EmbeddingDataPicker` is the only component that uses it, but I added it for safety anyway.